src/tamp_improv/benchmarks/blocks2d.py

```diff
@@ -77,10 +78,10 @@ def as_set(self) -> set[Predicate]:
         }
 
 
-class BaseBlocks2DSkill(LiftedOperatorSkill[NDArray[np.float32], NDArray[np.float32]]):
+class BaseBlocks2DSkill(LiftedOperatorSkill[GraphInstance, NDArray[np.float32]]):
```

Comment:
I think we should maintain backwards compatability here so that we can still run vector-based experiments in the future. Maybe it would make sense to create new subclasses that differ only in the graph related stuff?